src/mpi/init/init_dbg_logging.c

```diff
@@ -42,7 +42,7 @@ void MPII_init_dbg_logging(void)
 #ifdef MPICH_IS_THREADED
     MPL_dbg_init(NULL, NULL, TRUE, TRUE,
                  MPIR_Process.comm_parent != NULL, MPIR_Process.comm_world->rank,
-                 MPIR_ThreadInfo.isThreaded);
+                 MPIR_ThreadInfo.thread_provided == MPI_THREAD_MULTIPLE);
```

Comment:
Since `MPIR_ThreadInfo.thread_provided` is always defined, can we remove the preprocessor guards?

Reply:
I like the suggestion.

Reply:
Updated in commit https://github.com/pmodels/mpich/pull/4199/commits/96594d579ca73407a6d63443266acc731e1bc8b5